composer.json

```
@@ -23,7 +23,7 @@
   },
   "require": {
     "composer/installers": "^1.5",
-    "unzerdev/php-sdk": "1.1.4.0"
+    "unzerdev/php-sdk": "^1.1.4"
```

This version constraint will not exclude breaking changes.
It should be ~1.1.4 to make sure breaking changes of the SDK will always require a compatibility check and update of the Plugin.
```suggestion
    "unzerdev/php-sdk": "~1.1.4"
```